zkwebui/WEB-INF/src/org/adempiere/webui/apps/form/WInvoiceGenFromShipment.java

```diff
@@ -0,0 +1,236 @@
+/******************************************************************************
+ * Copyright (C) 2009 Low Heng Sin                                            *
+ * Copyright (C) 2009 Idalica Corporation                                     *
+ * This program is free software; you can redistribute it and/or modify it    *
+ * under the terms version 2 of the GNU General Public License as published   *

```

Comment:
Can you change license to **version 2 or later**?

Reply:
Hi @yamelsenih 
I think the NPE is solved with Pull Request #2060